cmd/clusterctl/cmd/init.go

```diff
@@ -103,6 +104,8 @@ func init() {
 		"IPAM providers and versions (e.g. infoblox:v0.0.1) to add to the management cluster.")
 	initCmd.PersistentFlags().StringSliceVar(&initOpts.runtimeExtensionProviders, "runtime-extension", nil,
 		"Runtime extension providers and versions (e.g. test:v0.0.1) to add to the management cluster.")
+	initCmd.PersistentFlags().StringSliceVar(&initOpts.addonProviders, "add-on", nil,
```

Comment:
```suggestion
	initCmd.PersistentFlags().StringSliceVar(&initOpts.addonProviders, "addon", nil,
```
To be consistent with the flags in other commands.